server/src/main/scala/org/ada/server/models/WidgetSpec.scala

```diff
@@ -44,6 +44,7 @@ case class DistributionWidgetSpec(
   relativeValues: Boolean = false,
   numericBinCount: Option[Int] = None, // TODO: rename to binCount
   useDateMonthBins: Boolean = false,
+  orderByField: Option[Boolean] = None,
```

Comment:
Also the attribute's name `orderByField` is rather ambiguous. What about something like `orderByLabels`?